### PR TITLE
Installation process with composer.

### PR DESCRIPTION
**Pull request by vpassama**
This PR is about automation for the installation process with composer.

One bundle is added : https://github.com/Cube-Solutions/composer-dist-installer
It provides questions/answers about the config.inc.php.dist file and generate config.inc.php at the end.

Minor changes : 
- composer.json has been transformed by dos2unix
- composer.phar is now ignored.

### CODE REVIEW

**Comment by vpassama** [comment]
I added "symfony/symfony": "~2.6.0" into amqpmttworkers' composer.json in order to prevent symfony 2.7 installation.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 /components/
 /logs/
 config.inc.php
+composer.lock
```
Comment (Netmisa) [inline .gitignore]: We prefer to keep composer.lock to deploy quickly the worker in new instance.

**Comment by Netmisa** [comment]
I tested this and for me it's ok.
We just need to add composer.lock and after we can merge it

thx @vpassama

**Comment by vpassama** [comment]
The PR has been updated. composer.lock is back.

**Comment by Netmisa** [comment]
I have two warnings when I do an install with composer

```
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. Run update to update them.
```

For this warning could you update it please ?

```
Class Cube\ComposerDistInstaller\Bootstrap is not autoloadable, can not call post-install-cmd script
```

Is it possible two resolve this ?

Thanks

---

EDIT: I just update the composer lock and all warnings was resolved. So you just need to update it

**Comment by vpassama** [comment]
composer.lock has been updated.

**Comment by datanel** [comment]
:ok_hand:

**Pull request merged**